MOBILE_CLIENT/ANDROID/_MEDIUM/PATH_TRAVERSAL/description.md

```diff
@@ -0,0 +1,33 @@
+Application can allows the attacker to navigate through the file system beyond the intended 
+boundaries. This can lead to unauthorized access to sensitive files or directories. 
+By manipulating file paths, an attacker can bypass access controls and retrieve or modify critical data, 
+such as configuration files, user credentials, or confidential documents. This vulnerability poses 
+a significant threat as it enables the attacker to escalate privileges, execute arbitrary code, 
+or launch further attacks on the system.
+
+
+Can expose the content of the passwords file using the following code:
+
+```python
```

Comment:
Why mark it as Python?

Reply:
Dart not supported. python is the most one close to dart in term of highlighting the keywords

Reply:
I can see it supported here:
https://highlightjs.org/static/demo/

You can use dart.